integreat_cms/cms/models/regions/region.py

```diff
@@ -321,6 +321,22 @@ class Region(AbstractBaseModel):
         ),
     )
 
+    deepl_usage_limit = models.PositiveIntegerField(
+        default=50_000,
+        blank=True,
+        verbose_name=_("Yearly DeepL machine translation budget"),
+        help_text=_("Number of words that can be machine translated by DeepL per year"),
+    )
+
+    deepl_usage_remaining = models.PositiveIntegerField(
+        default=50_000,
+        verbose_name=_("Remaining DeepL machine translation budget"),
+        blank=True,
+        help_text=_(
+            "Remaining number of words that can be translated by DeepL this year"
+        ),
+    )
```

Comment:
Instead of keeping track of the remaining words, I would rather start from 0 and count the consumed words. This makes it easier to change the limit independently from the used words.